File: sample/src/main/java/com/github/lany192/box/sample/activity/HelloActivity.java
package com.github.lany192.box.sample.activity;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.github.lany192.box.binding.BindingActivity;
import com.github.lany192.box.sample.databinding.ActivityHello2Binding;

public class HelloActivity extends BindingActivity<ActivityHello2Binding> {
    private boolean show;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding.button.setOnClickListener(v -> {
            if (show) {
                show = false;
                controller.setAppearanceLightStatusBars(true);
                controller.setAppearanceLightNavigationBars(false);
            } else {
                show = true;
                controller.setAppearanceLightStatusBars(false);
                controller.setAppearanceLightNavigationBars(true);
            }
        });
    }
}

File: sample/src/main/java/com/github/lany192/box/sample/activity/HelloActivity.kt
package com.github.lany192.box.sample.activity

import android.os.Bundle
import android.util.Log
import androidx.viewbinding.ViewBinding
import com.github.lany192.box.activity.ViewBindingActivity
import com.github.lany192.box.sample.databinding.ActivityHelloBinding
import com.github.lany192.box.view.ActivityBinding

class HelloActivity : ViewBindingActivity<ActivityHelloBinding>() {

    override fun init(savedInstanceState: Bundle?) {
        binding.getContent<ActivityHelloBinding>().showTextVew.text = "哈哈"
    }

    override fun getContentView(): ActivityHelloBinding {
        return ActivityHelloBinding.inflate(layoutInflater)
    }

    override fun init(savedInstanceState: Bundle?) {
        //
    }
}